lib/pbench/server/auth/auth.py

```diff
@@ -198,7 +207,8 @@ def verify_auth(auth_token: str) -> Optional[Union[User, InternalUser]]:
                 ActiveTokens.delete(auth_token)
             except Exception as e:
                 Auth.logger.error(
-                    "User passed expired token but we could not delete the token from the database. token: {!r}: {}",
+                    "User passed expired token but we could not delete the"
+                    " token from the database. token: {!r}: {}",
```

Comment:
While I really appreciate avoiding code lines which are "too wide for the screen", I'm reluctant to split strings which are printed or logged as diagnostic messages:  when we find one in the log, it's frustrating if searching the code doesn't turn it up because it was split.

I would suggest rephrasing instead:
```
                    "Error deleting expired token ({!r}) from the database: {}",
```
(but, I'll "pre-resolve" this comment).